subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r31/PersistentCompositeDependencySubstitutionCrossVersionSpec.groovy

```diff
@@ -102,7 +104,10 @@ class PersistentCompositeDependencySubstitutionCrossVersionSpec extends ToolingA
         ideaModule.dependencies.size() == 1
         with(ideaModule.dependencies.first()) {
             it instanceof IdeaModuleDependency
-            dependencyModule == null
+            // this currently fails for current -> 3.5
+            if (targetVersion < GradleVersion.version("4.0")) {
```

Comment:
@adammurdoch This is what I thought you meant, but does not work as I expected.  How do I alter behavior based on the version of the tooling API I'm running, as opposed to the version I'm running against?